test/include/autoinstall/AutoinstallAskInclude_test.rb

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env rspec
+
+root_path = File.expand_path('../..', __FILE__)
+ENV["Y2DIR"] = File.join(root_path, 'src')
+
+require "yast"
+require_relative "../../../src/include/autoinstall/ask.rb"
```

Comment:
Ok. I've rewritting this part so the dummy client is now responsible for loading those bits.